sink/src/main/java/com/spredfast/kafka/connect/s3/sink/S3SinkTask.java

```diff
@@ -121,7 +148,7 @@ public void put(Collection<SinkRecord> records) throws ConnectException {
 	public void flush(Map<TopicPartition, OffsetAndMetadata> offsets) throws ConnectException {
 		Metrics.StopTimer timer = metrics.time("flush", tags);
 
-		log.debug("{} flushing offsets", name());
+		log.info("{} flushing offsets", name());
```

Comment:
This should be backed down to debug. It was moved up to info just for testing's sake.